openedx/features/redhouse_features/admin_panel/api/v0/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
+
+
+class UpdateUserActiveStatusSerializer(serializers.ModelSerializer):
+    """
+    Serializer for active status change endpoint.
+    """
+
+    class Meta:
+        model = User
+        fields = ('id', 'is_active')
```

Comment:
Instead of making specific serializers for specific fields, how about making some generic serializer (wherever possible and seems beneficial). e.g. in this case, we can make a UserSerializer that will return user's details including the status. What do you think?

Reply:
@danialmalik I think right now sending unwanted data here will increase the API response time in the long run. We can tailor this API in the future if needed. But if there is something in pipeline that needs the UserSerializer then we should definitely go for it.  Let me know what you think of this.

Reply:
I can think of one scenario in which we want at least email and username of current user, but that will be achieved by `request.user` object. 
The other scenario includes getting list of users detail so we will definitely have to make a new serializer and view I guess.